integration-tests/15_achievements.ts

```diff
@@ -0,0 +1,1043 @@
+import { createNewPupil, createNewStudent, pupilTwo, studentOne } from './01_user';
+import { test } from './base';
+import { v4 as generateUUID } from 'uuid';
+import { screenerOne } from './02_screening';
+import { adminClient } from './base/clients';
+import { prisma } from '../common/prisma';
+import { achievement_template_for_enum, achievement_type_enum, achievement_action_type_enum, lecture_appointmenttype_enum } from '@prisma/client';
+import { User, getUser } from '../common/user';
+import { getLogger } from '../common/logger/logger';
+import { Match } from '../graphql/generated';
+import { _createFixedToken } from '../common/secret/token';
+import assert from 'assert';
+
+void test('Reward student onboarding achievement sequence', async () => {
+    await adminClient.request(`mutation ResetRateLimits { _resetRateLimits }`);
+    await createStudentOnboardingTemplates();
+
+    // Verify Email
+    const { student, client } = await createNewStudent();
+    const user = await getUser(student.userID);
+
+    const student_onboarding_1 = await prisma.user_achievement.findFirst({
```

Comment:
Why are all these variables underscored and not camel cased?

Reply:
nope, and I changed it to restore consistency.